rasterio/crs.pyx

```diff
@@ -273,6 +273,24 @@ cdef class CRS:
             units_b = units_c
             return (units_b.decode('utf-8'), factor)
 
+    def to_geodetic(self):
+        """
+        Returns the geographic CRS but 
+        with traditional long/lat,easting/northing ordering
+
+        """
+        cdef CRS obj = CRS.__new__(CRS)
+        try:
+            # TODO can GeogCS be null/empty?
+            obj._osr = exc_wrap_pointer(OSRCloneGeogCS(self._osr))
+        except CPLE_BaseError as exc:
+            raise CRSError("The Geodetic CRS for this CRS could not be found: {}".format(exc))
+        finally:
```

Comment:
Let's make this an `else` clause, yes? Returning from a `finally` is a little bit undefined because that clause is supposed to execute no matter what exceptions may be raised.